{
  PrestoException exception=new PrestoException(new TestErrorCode(),"test");
  assertEquals(exception.getMessage(),"test");
  exception=new PrestoException(new TestErrorCode(),new RuntimeException("test2"));
  assertEquals(exception.getMessage(),"test2");
  exception=new PrestoException(new TestErrorCode(),new RuntimeException());
  assertEquals(exception.getMessage(),"test");
}
